Assets/MixedRealityToolkit/AudioManager/AudioManager.cs

```diff
@@ -0,0 +1,310 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Audio
+{
+    /// <summary>
+    /// The manager that handles the playback of AudioEvents
+    /// </summary>
+    public class AudioManager : MonoBehaviour
```

Comment:
Do you anticipate a per-scene audio manager instance or per-app?

Reply:
Definitely per app. You don't need it in the scene because it will add itself the first time an audio event is played.

Reply:
If it's not something we need to have in the scene, we may want to make this a system with a configuration profile with audio banks etc

Reply:
I would love for it to not have to be in the scene, but something needs to run the update function for the active events. If there's a better way than adding an object with the AudioManager script to the scene, I'm all for it!

Reply:
If you have time, we should sit and discuss. It might make sense to make the audio manager a service and it might make sense to keep it as-is.

Reply:
Based on how we've used this manager in our projects a service would make the most sense IMO.

Reply:
Yeah, I think we'll def need to servicily it and likely needs to be part of the core systems.